sharepoint/sharepoint-ps/sharepoint-online/Get-SPOMalwareFile.md

```diff
@@ -0,0 +1,65 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://docs.microsoft.com/powershell/module/sharepoint-online/Get-SPOMalwareFile
+applicable: SharePoint Online
+title: Get-SPOMalwareFile
+schema: 2.0.0
+author: vinandi
+ms.author: vinandi
+ms.reviewer:
+---
+
+# Get-SPOMalwareFile
+
+## SYNOPSIS
+
+Extracts and displays the malware related information of an infected file stored in SharePoint.
+
+## SYNTAX
+
+```powershell
+Get-SPOMalwareFile -FileUri <String> [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+The `Get-SPOMalwareFile` cmdlet runs on a single file. If the file is malware-infected then it returns the siteURL, filePath, malwareStatus flag and malwareInfo (if applicable). You must be a SharePoint Online administrator or Global Administrator to run the `Get-SPOMalwareFile` cmdlet. Note that this cmdlet does not work on files that are not malware-infected.

```

Comment:
>siteURL, filePath, malwareStatus flag and malwareInfo [](http://example.com/codeflow?start=107&length=53)

since since are the properties of the returned object, I'm not sure if they need to be called differently?

Reply:
Since the customer doesn't know about the SPOMalwareFile object, I have mentioned the object type and the properties explicitly.